clients/gtest/rocblas_gtest_main.cpp

```diff
@@ -209,10 +230,15 @@ static void rocblas_test_sigaction()
     act.sa_flags   = 0;
     act.sa_handler = rocblas_test_signal_handler;
 
-    for(int sig : {SIGABRT, SIGBUS, SIGFPE, SIGILL, SIGPIPE, SIGSEGV, SIGSYS, SIGUSR1, SIGUSR2})
-    {
+    for(int sig : {SIGABRT, SIGBUS, SIGFPE, SIGILL, SIGALRM, SIGSEGV, SIGSYS, SIGUSR1, SIGUSR2})
```

Comment:
`SIGPIPE` is was replaced with `SIGALRM` because `SIGPIPE` is asynchronous and we don't need it, while we need `SIGALRM`